src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -964,22 +952,22 @@ public static GenotypesContext updatePLsAndAD(final VariantContext selectedVC, f
      * @param allelesToUse     the new (sub)set of alleles to use
      * @return a new non-null GenotypesContext
      */
-    static private GenotypesContext fixGenotypesFromSubsettedAlleles(final GenotypesContext originalGs, final VariantContext originalVC, final List<Allele> allelesToUse) {
+    private static GenotypesContext fixGenotypesFromSubsettedAlleles(final GenotypesContext originalGs, final VariantContext originalVC, final List<Allele> allelesToUse) {
 
         // we need to determine which of the alternate alleles (and hence the likelihoods) to use and carry forward
-        final List<Integer> likelihoodIndexesToUse = determineLikelihoodIndexesToUse(originalVC, allelesToUse);
+        final List<List<Integer>> genpotypesLikelihoodIndexesToUse = determineGenotypesLikelihoodIndexesToUse(originalVC, allelesToUse);
 
         // create the new genotypes
-        return createGenotypesWithSubsettedLikelihoods(originalGs, originalVC, allelesToUse, likelihoodIndexesToUse, GenotypeAssignmentMethod.DO_NOT_ASSIGN_GENOTYPES);
+        return createGenotypesWithSubsettedLikelihoods(originalGs, originalVC, allelesToUse, genpotypesLikelihoodIndexesToUse, GenotypeAssignmentMethod.DO_NOT_ASSIGN_GENOTYPES);
     }
     /**
-     * Figure out which likelihood indexes to use for a selected down set of alleles
+     * Determine which likelihood indexes to use for each genotype for a subset of alleles
      *
      * @param originalVC        the original VariantContext
      * @param allelesToUse      the subset of alleles to use
-     * @return a list of PL indexes to use or null if none
+     * @return likelihood indexes for each genotype
```

Comment:
this method needs better documentation if what the returned value is - what is this list of lists of?